Svc/Interfaces/SpiInterface.fppi

```diff
@@ -0,0 +1,2 @@
+@ Port to perform a synchronous read/write operation over the SPI bus
```

Comment:
This interface should go in `Drv/Interfaces`.

If an interface is used in service it goes in `Svc/Interfaces`, if it is used in drivers `Drv/Interfaces`, if it is for the whole framework `Fw/Interfaces`.